### DISCUSSION TITLE
Website link button in a dashboard

### DISCUSSION THREAD

**Discussion by michalutko** [joonaspaakko/ScriptUI-Dialog-Builder-Joonas#98] [Q&A]
Awesome product!! I love how easy it is to build and customize the dashboards. Now for the stupid question.

How do you go about linking to a website from a dashboard. I would like to have a help button that would lead a person to a resource on a website.
 
Sorry for wasting your time.

**Comment by joonaspaakko** [comment]
I suppose SDB could have a premade item that opens links, but it doesn't really try to solve edge cases, rather just offer basic items to build layouts.

How you'd do this first of all probably depends on whether or not you're using ScriptUI or CEP. In CEP I'd imagine a simple link should work, but to my knowledge SUI is a mixed bag in that something might work in one app but not in another (also older versions might not work with the same methods). I don't think I've ever attempted to add an external link in a SUI dialog, so I can't really give an answer that I know would work.

[This answer in the adobe forums](https://community.adobe.com/t5/photoshop-ecosystem-discussions/open-web-url-from-script-ui-in-default-browser/m-p/11076765#M325184) seems like a pretty smart SUI workaround (at least for Photoshop). I saw someone saying it didn't work in some other adobe app, but I feel like writing a text file and then executing (opening) it should work in all apps and even like 15 year old versions. But like I said I haven't tried doing it myself.

> It's fairly self explanatory, but it looks like it creates a html file on the fly, which has a redirect to the desired url and then the script tries to open the html file, which should force the default application (usually a browser) to open the file/the web page you're redirecting to. But of course if someone has set a code editor or anything else as the default application for .html files, the file wouldn't open in the browser.
